tests/threads_detach.rs

```diff
@@ -0,0 +1,59 @@
+#![cfg(feature = "invocation")]
+extern crate jni;
+extern crate error_chain;
+
+use std::thread::{spawn, sleep};
+
+use jni::{
+    objects::JValue,
+    sys::jint,
+};
+
+mod util;
+use util::jvm;
+use std::time::Duration;
+
+// We forced to combine several tests in one function, because every test function is running in
```

Comment:
Does cargo support 'single threaded' mode for a single test file via an annotation?

If it doesn't, I'd consider either splitting into indy files (that would hurt navigation/readability, but provide perfect isolation and better reporting as individual tests), or moving them all in a single file in separate test methods. Currently a single test is in vm.rs, three — in here, which makes little sense.

Reply:
No support afaik, so I split these tests into several files, yes.